src/main/webapp/app/detail-overview-list/detail-overview-list.component.html

```diff
@@ -19,13 +19,7 @@ <h3 class="section-headline" [id]="headlinesRecord[section.headline]">{{ section
                         <div jhiExerciseDetail [detail]="detail"></div>
                     }
                     @case (DetailType.Date) {
-                        <dd id="detail-value-{{ detail.title }}">
-                            @if (detail.data.date) {
-                                {{ detail.data.date | artemisDate }}
-                            } @else {
-                                <ng-container *ngTemplateOutlet="noData" />
-                            }
-                        </dd>
+                        <div jhiExerciseDetail [detail]="detail"></div>
```

Comment:
Why is this wrapped in a div instead of being an html-element itself? (<jhi-…>)

Reply:
Usually, directives are assigned to elements via an attribute (here `selector: '[jhiExerciseDetail]',` ) because this is more flexible (we can just assign it to any element) than other approaches. 
However, here we only want to get the proper element returned, so we can change the selector - I have no strong opinion in this particular case - an element selector might even be the more reasonable approach

Reply:
I adjusted it to an element selector - I guess that should be fine for this directive

Reply:
Since Codacy does not seem to be happy with using an element selector, and I did not find a way to deactivate it I changed it back to the attribute selector to prevent that the style and best practice checks fail when somebody touches this component in the future